Electrino/Electrino/ENOJavaScriptApp.m

```diff
@@ -88,6 +126,18 @@ - (id)init
     return self;
 }
 
+// Create a new context for just evaluating the file
+// ISSUE: JSContext does not include -copyWithZone: method, so we have to manually copy the required methods.
+-(JSContext*)newContextForEvaluation
+{
+	JSContext* newContext = [[JSContext alloc] initWithVirtualMachine:self.jsVM];
+	newContext[@"require"] = self.jsContext[@"require"];
+	newContext[@"process"] = self.jsContext[@"process"];
+	newContext[@"console"] = self.jsContext[@"console"];
+	[newContext evaluateScript:@"var exports = {};"]; // Evaluated so the developer doesnt have to
```

Comment:
Sorry, my mistake. It should be

```obc
newContext[@"exports"] = module.exports;
```

I've changed it in the example. I'll try to help more on this later.